ui/overview_page.go

```diff
@@ -639,7 +640,16 @@ func (pg *overviewPage) walletSyncBox(gtx layout.Context, inset layout.Inset, de
 	})
 }
 
-func (pg *overviewPage) Handler(eq event.Queue, c pageCommon) {
+func (pg *overviewPage) Handler(eq event.Queue, c pageCommon, win *Window) {
+
+	if win.wallet != nil {
+		isDarkModeOn := win.wallet.ReadBoolConfigValueForKey("isDarkModeOn")
+		if isDarkModeOn != win.theme.DarkMode {
+			win.theme.SwitchDarkMode(isDarkModeOn)
```

Comment:
Why is this method called in the overview and settings page. Shouldn't it be called just once from the settings page?

Reply:
I called it in the settings page to handle the button click event dark mode. In the overview page because it is the first page loaded when app startup, i can read config from the wallet in this page and switch to the dark mode.